### ISSUE TITLE
No license terms

### ISSUE THREAD

**Issue by nonk123** [kgsws/doom_ace#3]
TL;DR: 🤓

I don't mind the legal implications of this since I'm no lawyer, but it makes sense to license your work anyway. No license terms means e.g. my fork would break the copyright laws of a country or two (possibly my country of residence too?). Also, if you decide to add any dependencies (like I did in my fork and realized only now), it's a bit of a grey zone whether you're allowed to use them due to them having their own license terms.

My proposal: switch to GPLv3. The underlying Doom executable might be able to pass as a "system library", meaning it's compatible with the license. GPLv3 is also [compatible with a lot of other licenses](https://www.gnu.org/licenses/license-list.html#GPLCompatibleLicenses) such as MIT, BSD 3-Clause, or Apache 2.0. This just means it's possible to use dependencies licensed under those conditions without copyright infringement. Lastly, this resolves all the copyright/distribution ambiguities I mentioned earlier.

**Comment by nonk123** [comment]
It's easy enough if you follow the instructions on [Choose a License](https://choosealicense.com/licenses/gpl-3.0/).
